docs/REST API/rest-api.md

```diff
@@ -8182,24 +8205,24 @@ Content-Length: <length>
 Body: <blob>
 ```
 
-### Delete software
+### Delete software available for install
 
 _Available in Fleet Premium._
 
-Delete a software package.
+Deletes software that's available for install (package or App Store app).
 
-`DELETE /api/v1/fleet/software/titles/:software_title_id/package`
+`DELETE /api/v1/fleet/software/titles/:software_title_id/available_for_install`
```

Comment:
Rename `DELETE /api/v1/fleet/software/titles/:software_title_id/package`. Use one API endpoint to remove uploaded package and App Store app.